static/css/main.css

```diff
@@ -136,12 +136,13 @@ div.content {
 }
 
 .list {
-   border-style: double;
-   list-style-type:none;
-   padding:8px;
-   margin:0
-   border-bottom:1px solid #ddd
-   font-size     : 14pt;
+    width: 800px;
```

Comment:
In testing this change it looks like we end up with huge table cells with one ingredient in each. maybe update list to the following? 

`.list {
    list-style-type:none;
    padding:8px;
    border-bottom:1px solid #ddd
    font-size     : 14pt;
    overflow: scroll;
    overflow-x: hidden;
    margin: auto;
    border-style: solid;
}`